src/views/keyevents/Keybindings.ts

```diff
@@ -124,14 +125,26 @@ export const KeybindingsForMenu: KeybindingMenuType[] = [
     },
     {
         action: KeyboardAction.tabPrevious,
-        accelerator: "CmdOrCtrl+K",
+        accelerator: "Ctrl+Shift+Tab",
     },
     {
         action: KeyboardAction.tabNext,
-        accelerator: "CmdOrCtrl+J",
+        accelerator: "Ctrl+Tab",
     },
     {
         action: KeyboardAction.tabClose,
+        accelerator: "CmdOrCtrl+Shift++W",
```

Comment:
The standard in Mac apps seems to be Cmd+Shift+W closes a window, not a tab. (try it in Chrome, Sublime, iTerm, etc.) `black-screen` has both tabs and panes so it's a bit more complicated, but I still think Cmd+Shift+W should be reserved for closing a window.